### PR TITLE
improvements for everything Azure SQL

### PR DESCRIPTION
**Pull request by yorek**

### CODE REVIEW

**Comment by milanm** [comment]
Looks good!

File: README.md [hunk 1]
```diff
@@ -201,7 +201,7 @@ If you have any suggestions or missing products, create a pull request on this R
 * [**Azure Arc**](https://azure.microsoft.com/en-gb/services/azure-arc/): Secure, develop, and operate infrastructure, apps, and Azure services anywhere.
 * [**Azure DevOps**](https://azure.microsoft.com/en-gb/services/devops/): Services for teams to share code, track work and ship software.
 * [**Azure IoT Edge**](https://azure.microsoft.com/en-gb/services/iot-edge/): Extend cloud intelligence and analytics to edge devices.
-* [**Azure SQL Database**](https://azure.microsoft.com/en-gb/products/azure-sql/database/): Managed, intelligent SQL in the cloud.
+* [**Azure SQL Database**](https://azure.microsoft.com/en-gb/products/azure-sql/database/): Small-footprint, edge-optimised SQL database engine with built-in AI.
```
Comment (milanm) [inline README.md:204]: Should it be edge-optimised? This is Azure SQL Database, not Azure SQL Edge ?

Comment (yorek) [reply README.md:204]: Yeah, my bad, the description should be "Managed, intelligent SQL database in the cloud."

Comment (yorek) [reply README.md:204]: Yeah, my bad, the description should be "Managed, intelligent SQL database in the cloud."

**Pull request merged**